### PR TITLE
Overrides

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@isoos – I landed this, but would still like your comments...

File: lib/pana.dart [hunk 1]
```diff
@@ -142,7 +142,14 @@ class PackageAnalyzer {
 
     if (summary != null) {
       try {
-        libraryScanner = new LibraryScanner(pkgDir, isFlutter);
+        var overrides = {
```
Comment (isoos) [inline lib/pana.dart]: why are these overrides?

Comment (kevmoo) [inline lib/pana.dart]: They import dart:io – but in a way that's safe for the web.

Comment (isoos) [inline lib/pana.dart]: We can already detect that in a generic way, not specific to any given package. Do we need something over that?

**Pull request merged**